the-new-octal/src/app/common/loading-animation-colors/loading-animation-colors.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-loading-animation-colors',
+  templateUrl: './loading-animation-colors.component.html',
+  styleUrls: ['./loading-animation-colors.component.scss']
+})
+export class LoadingAnimationColorsComponent implements OnInit {
+
+  @Input() list$ = new Observable;
```

Comment:
2x kérjük le ugyanazt a végpontot ugyanazt az eredményt kapva.
Egy esetlegesség miatt jelenik meg az adat később.

Vagy legyen @Input @Output és a list az legyen Output.
Vagy akár egy ng-container-be csinálhatod az `(list$ | async) as list`-et amit megkap ez a komponens és a későbbi `*ngFor`

![image](https://user-images.githubusercontent.com/10902799/154828744-6db2ff77-6bfd-4434-ade0-38b86f8d6a5e.png)